src/studio-home/data/thunks.js

```diff
@@ -30,7 +30,7 @@ function fetchStudioHomeData(search, hasHomeData) {
       }
     }
     try {
-      const coursesData = await getStudioHomeCourses(search || '');
+      const coursesData = await getStudioHomeCourses(search || '', customParams);
```

Comment:
`customParams` always will have a value? If not, better do:

```suggestion
      const coursesData = await getStudioHomeCourses(search || '', customParams = {});
```

Reply:
I would suggest to leave it, just in case.